api/src/models/publication.model.js

```diff
@@ -42,7 +42,23 @@ const publicationSchema = new mongoose.Schema({
     },
     yearPublished: {
         type: Date
-    }
+    },
+    category: {
+        type: {
+            type: String,
+            enum: ['CONFERENCE', 'JOURNAL'],
+            required: true
+        },
+        categoryTitle: {
```

Comment:
What's the difference between `categoryTitle` and the enums defined for category `type`?

Reply:
![image](https://user-images.githubusercontent.com/17608976/118254379-61be2700-b4ee-11eb-9493-8bd5ec9c24a8.png)
Sorry for the bad handwriting.

Reply:
Got it, thanks!